ls1intum/Artemis
# Center nav-buttons in exams

### Checklist

### Motivation and Context


The buttons to navigate in an exam are not centered and sometimes 'jumped' when navigating.

### Description

Rearranged the layout so that the buttons are centered and do not 'jump'.

### Steps for Testing


1. Log in to Artemis
2. Create and prepare an exam with some exercises
3. Participate in the exam and navigate trough the exercises
4. Check that the navigation is in the middle and doesn't 'jump'

### Screenshots


<img width="1421" alt="Bildschirmfoto 2020-07-12 um 21 43 46" src="https://user-images.githubusercontent.com/51987021/87255204-ee402900-c488-11ea-9df2-40cc78ae0b53.png">
